doc/observability-deployment.md

```diff
@@ -3,7 +3,7 @@
 # Observability with Open Liberty
 
 
-The following document covers various topics for configuring and integrating your Open Liberty runtime with monitoring tools in the OpenShift cluster.
+The following document covers various topics for configuring and integrating your Open Liberty runtime with monitoring tools in the OpenShift cluster. This document has been tested with Red Hat OpenShift Container Platform (RHOCP) 4.2.
```

Comment:
this should be a new file, called observability-deployment-rhocp4.2.md.  existing file should be renamed to observability-deployment-okd3.11.md